relations/services/get.py

```diff
@@ -1,13 +1,51 @@
 """Service for relation retrlieval."""
-from relations.domain import Relation, RelationID
+from relations.domain import EPrint, Resource, Relation, RelationID, \
+    RelationType
+from .model import db, RelationDB
 
 
 class NotFoundError(Exception):
     """Error that the retrieved relation cannot be found."""
 
-    pass  # not implemented yet
+    pass
+
+
+class DBLookUpError(Exception):
+    """Error that happens in lookup."""
+
+    pass
 
 
 def from_id(id: RelationID) -> Relation:
     """Retrieve a relation from an ID."""
-    pass  # not implemented yet
+    # query to DB
+    try:
+        rels = db.session.query(RelationDB) \
+            .filter(RelationDB.id == int(id)) \
+            .limit(1) \
+            .all()
```

Comment:
Might be more concise to use [``.first()``](https://docs.sqlalchemy.org/en/13/orm/query.html#sqlalchemy.orm.query.Query.first) here -- should generate the same or similar underlying SQL. Returns ``None`` if there is no result.

Reply:
I rewrote it with `.one()` in another PR, actually.

Reply:
So I will leave it as it is. We'll change it in another PR.